### DISCUSSION TITLE
Multiplication Game Screen Flow

### DISCUSSION THREAD

**Discussion by Pertempto** [software-factory-collective/mabocados#14] [Design]
![2024-11-14-02-58-16-469.jpg](https://github.com/user-attachments/assets/a265b6bd-8057-4e51-bdf5-5981936e6427)

**Comment by Pertempto** [comment]
https://icon-sets.iconify.design/?query=avocado

There are 31 avocado icons to choose from. 😊

**Comment by Pertempto** [comment]
This icon set seems nice


https://icon-sets.iconify.design/hugeicons


It is MIT licensed and seems to have a good variety of icons (3929 total)

I like how the avocado icon is well defined in this set

**Reply by LeanMendez** [reply]
I like this set too.
It's nice to find a MIT licensed icon set that has so many icons. 
I usually use icons with thinner borders, but for games and things that are likely to be used by children, this set fits perfectly.

**Reply by Pertempto** [reply]
I agree!

**Comment by Pertempto** [comment]
The proposed screen flow does not take into account levels. I think that is OK for the initial demo
